release_creation/main.py

```diff
@@ -38,33 +37,45 @@ def main():
     args = parser.parse_args()
     version = args.input_version
     operation = args.operation
-    latest_version, is_draft, latest_release = get_latest_bundle_release(version)
-    logger.info(f"Retrieved latest version: {latest_version} "
-                f"and latest release: {latest_release.tag_name if latest_release else None}")
+    if version.startswith("0.0"):
+        latest_version = Version.coerce(DEV_VERSION)
+        is_draft = True
```

Comment:
Does this need to be set?  In `create_dev_bundle` you have draft hardcoded to false.

Reply:
This should actually be hardcoded to false as we do a check later for draft is true